lib/pegboard/manager/src/main.rs

```diff
@@ -1,35 +1,63 @@
+use std::path::Path;
+
 use anyhow::*;
 use futures_util::StreamExt;
 use tracing_subscriber::prelude::*;
 use url::Url;
 use uuid::Uuid;
 
-mod client;
 mod container;
 mod ctx;
 mod utils;
 
-use client::Client;
+use ctx::Ctx;
+
+const PROTOCOL_VERSION: u16 = 1;
 
 #[tokio::main]
```

Comment:
should we use a current threaded scheduler or limit the threads to 2? this is currently multithreaded, will spin up a lot of excess threads.

Reply:
we do the same in the toolchain

Reply:
idk how much parallelism we'll actually need